core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponent.scala

```diff
@@ -121,6 +123,17 @@ trait RawlsBillingProjectComponent {
       }.toMap)
     }
 
+    def setBillingProjectExport(projectName: RawlsBillingProjectName, datasetName: Option[String], tableName: Option[String]): WriteAction[Int] = {
+      rawlsBillingProjectQuery
+        .filter(_.projectName === projectName.value)
+        .map(bp => (bp.spendReportDataset, bp.spendReportTable))
+        .update(datasetName, tableName)
+    }
+
+    def clearBillingProjectExport(projectName: RawlsBillingProjectName): WriteAction[Int] = {
+      setBillingProjectExport(projectName, None, None)
+    }
+
```

Comment:
ah yeah, I decided part-way through that I was going to start calling these things "spend report configurations" since the "export" terminology in the ticket was pretty confusing. I'll harmonize things across the board 👍